src/cow-react/modules/application/containers/App/index.tsx

```diff
@@ -43,6 +52,9 @@ export function App() {
         </styledEl.BodyWrapper>
         <styledEl.FooterWrapper>
           <Footer />
+
+          {/* Winter Theme */}
+          <Snowfall color={transparentize(0.7, theme.text1)} snowflakeCount={50} />
```

Comment:
uber-nitpick, but if you  create your own component, you can do minimal changes in the `App/index.tsx`. This way, instead of reverting the changes when chrismas is over, we can just remove the new component and leave it in the project for next year?

Reply:
Yes good suggestion, will create a separate PR for this.